src/expectSaga/matchers/index.js

```diff
@@ -35,6 +37,9 @@ cps.fn = fn => cps.like({ fn });
 fork.like = like('fork');
 fork.fn = fn => fork.like({ fn });
 
+getContext.like = like('getContext');
+getContext.property = property => getContext.like(property);
```

Comment:
Since `getContext` already accepts the property name as its only argument, `getContext.like` and `getContext.property` seem redundant. Can you remove them please?